docs/roadmap/phases.mdx

```diff
@@ -52,14 +52,14 @@ Explanation of what they are, why they matter, and how they work goes here.
 #### Payout Strategies
 
 There are 2 main strategies for SourceCred payouts: 
-- **Balanced:** looks at everyone's total lifetime XP, and will payout the dif between the person's XP/SEEDs (incentivizes exploration of new things that might not be immediately obvious). 
+- **Balanced:** makes sure that everyone's total lifetime payed-out SEEDs matches everyone's total lifetime XP (incentivizes exploration of new things that might not be immediately obvious).
 - **Immediate:** proportional amount of XP you created that week (incentivizes exploitation of current rewards). 
 
 SourceCred uses an 80/20 strategy for Grain distributions (their version of XP): 
 - 80% from the balanced payout system
 - 20% from the immediate payout system.
 
-It's important to have both. If you only use balanced it would also create immediate rewards because new payouts have a huge delta between SEEDs (0) and XP (X), but... if there's a slow week those who keep contributing would actually get less XP in the future as more people start to contribute (probably due to the efforts of the people who contributed when no one was around). We need to incentivize people to contribute all the time, esp when no one else is contributing, so we need the immediate payouts as well. If we have both "balanced and immediate" payouts then people will get a bonus for contributing when no one else does.
+It's important to have both. If you only use balanced it would also create immediate rewards because new payouts have a huge delta between SEEDs (0) `Why 0? Isn't that related to the lifetime payout (https://github.com/sourcecred/sourcecred/blob/1abed5f0edffbc262ca726e3d0640a58e2daccc8/src/grain/createGrainAllocation.js#L230)?` and XP (X), but... if there's a slow week those who keep contributing would actually get less XP in the future as more people start to contribute (probably due to the efforts of the people who contributed when no one was around). We need to incentivize people to contribute all the time, esp when no one else is contributing, so we need the immediate payouts as well. If we have both "balanced and immediate" payouts then people will get a bonus for contributing when no one else does.
```

Comment:
If someone is a new contributor, the "earned" would be 0 while the target (based on XP earned that week), is non 0.

Reply:
Okay, based on this I updated the wording of this paragraph.